cmd/uplink/cmd/setup.go

```diff
@@ -55,5 +59,53 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	return process.SaveConfigWithAllDefaults(cmd.Flags(), filepath.Join(setupDir, "config.yaml"), nil)
+	var override map[string]interface{}
+	if setupCfg.Interactive {
+		fmt.Print("Enter your Satellite address: ")
```

Comment:
I think we should check if this config has been provided as a flag by the user, and not prompt for it in such case.

This way if the user passes all required config as flags, they won't need to pass `--interactive=false`.